src/ivc/incrementally_verifiable_computation.rs

```diff
@@ -112,4 +240,49 @@ where
         // TODO #31
         todo!("Logic at `RecursiveSNARK::verify`")
     }
+
+    fn prepare_primary_td<const T: usize, RP1>(
+        k_table_size: u32,
+        instance: [C1::Scalar; 2],
+    ) -> (TableData<C1::Scalar>, StepConfig<A1, C1::Scalar, SC1, T>)
+    where
+        RP1: ROPair<C1::Scalar, Config = MainGateConfig<T>>,
+    {
+        let mut primary_td = TableData::new(k_table_size, instance.to_vec());
+        let config = primary_td.prepare_assembly(
```

Comment:
Add a TODO to link to #146 

prepare_assembly contains all circuit info except it missing the fixed column and selectors values which need to be hashed into pp_public_hash to be secure.